bazel/extensions/extensions_build_config.bzl

```diff
@@ -88,6 +87,7 @@ EXTENSIONS = {
     #
 
     "envoy.matching.inputs.application_protocol":       "//source/extensions/matching/network/application_protocol:config",
+    # Ideally these would be split up. We'll do so if anyone cares.
```

Comment:
this looks fine to me, but let's chat about it. to be completely honest, i'm not 100% sure exactly how to review the changes in this file.

Reply:
Process was I copied over things and then commented out new things that we dnot think we are going to use

Reply:
sounds good. i read [this section](https://github.com/envoyproxy/envoy/blob/main/bazel/README.md#enabling-and-disabling-extensions) over to better understand what we're doing here and i think all the changes seem reasonable. thanks!